ios/Mapswipe/Info.plist

```diff
@@ -15,11 +15,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.1.0</string>
+	<string>1.1.1</string>
```

Comment:
Do you mind bumping this to 1.2.2. I bumped up to a minor version rather than a patch version with the OSS updates here: https://github.com/mapswipe/mapswipe/blob/baab45dd15534bbc292d6acc8728a09c46822bc7/android/app/build.gradle#L87-L88

We'll also need to increment the version number in build.gradle as well as increment the `versionCode`. I'm happy to increment the Android parts as needed and I'll probably need to do that before we merge anyway so we get the same version going.

Reply:
I've been meaning to do something like this https://medium.com/@andr3wjack/versioning-react-native-apps-407469707661 since I think it would simplify this a little bit